### PR TITLE
feat: move test strategy and test plan to the GitHub tm-188

### PR DESCRIPTION
**Pull request by MariiaKrasylnykova**

### CODE REVIEW

File: docs/Test_Plan.md
Comment (what1s1ove) [inline docs/Test_Plan.md:1]: all files should be in a `kebab-case` notation

File: docs/Test_Plan.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+**BSWA 2023-2024**
+**[TrackMates](http://development.eu-central-1.elasticbeanstalk.com/)**
+**Test Plan**
+<br/>
```
Comment (what1s1ove) [inline docs/Test_Plan.md:4]:
```suggestion

```
you can use new line and it will be as a `br` tag

File: docs/Test_Plan.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+# Introduction
+
+The purpose of this test plan is to describe the testing process of the “TrackMates” web and Progressive Web App (PWA). “TrackMates” is an application for habit tracking in online courses, providing users with a platform to manage, track, and share their learning progress.
+
+# 1. Feature priorities
```
Comment (what1s1ove) [inline docs/Test_Plan.md:10]: h1 (`#`) can be only one in the document

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: docs/Test_Strategy.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+| Risk                           | Action point                                                                                                                                       | Responsible |
+| ------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| Absence of team member         | Provision has to be made for the temporary assignment of duties to another competent specialist. Correct planning of all processes                 | Coaches     |
+| Your absence                   | To inform the team about your absence as soon as possible in Slack, provide information about your urgent tasks                                    | QA          |
+| Faulty test results            | Implement peer reviews and validation checks to verify the accuracy of test results before finalizing them. Analyze a problem, create a bug report | QA Team     |
+| Test environment not available | Inform JS Couch on Slack, execute the testing process locally                                                                                      | JS Coach    |
```
Comment (fshabanov) [inline docs/Test_Strategy.md:261]:
```suggestion
| Test environment not available | Inform JS Coach on Slack, execute the testing process locally                                                                                      | JS Coach    |
```

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: docs/Test_Strategy.md (hunk 3 above)
Comment (MariiaKrasylnykova) [reply docs/Test_Strategy.md:261]: Sorry for this typo

File: docs/Test_Plan.md (hunk 2 above)
Comment (MariiaKrasylnykova) [reply docs/Test_Plan.md:10]: Got all your remarks, thanks

File: docs/test-strategy.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+| Not enough time for testing    | Prioritize testing activities based on critical functionalities and allocate additional resources or extend timelines if necessary                 | QA Team     |
+
+### Project Implementation Risks:
+
+- Technical Complexity: Developing a web and Progressive Web App (PWA) may introduce technical challenges, such as ensuring cross-browser compatibility, offline functionality, and responsiveness across various devices
+- Resource Allocation: Short project timeframe requires efficient resource allocation and coordination to meet deadlines and deliver high-quality results
+
+## 8. Test Plan
+
+**[Test Plan](https://docs.google.com/document/d/1nDmqEXj8gObbec4OoH-uJWdoxemDeMi_a_stn9vAZEI/edit?usp=sharing)** is composed by the QA team and contains:
```
Comment (fshabanov) [inline docs/test-strategy.md:267]: Should it be a link to the `test-plan.md` file?

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

File: docs/test-strategy.md (hunk 4 above)
Comment (MariiaKrasylnykova) [reply docs/test-strategy.md:267]: Good point, thanks, fixed it

File: docs/test-strategy.md [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+
+[Design (Desktop)](https://www.figma.com/file/P8QzgCZEaN0UU9QpjdggRW/BSA_TrackMates?type=design&node-id=0%3A1&mode=design&t=isWb7Jwsid5AxTL1-1)
+
+[Design (Mobile)](https://www.figma.com/file/JJjwl3joVKOwS0tzzYs8Vz/TrackMates?type=design&node-id=0%3A1&mode=design&t=ZuqmB1birN3911IH-1)
+
+[Specification](https://docs.google.com/document/d/1Vum3p6bc_2BQ_t3YaJbaH1We7UoZx7Hztgm0kTCeEe4/edit?usp=sharing)
+
+[Mind Map](https://miro.com/app/board/uXjVNuSbBvw=/?share_link_id=272684492193)
+
+[Test Plan](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/blob/main/docs/test-plan.md)
```
Comment (fshabanov) [inline docs/test-strategy.md:54]:
```suggestion
[Test Plan](./test-plan.md)
```
This should work

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by fshabanov — APPROVED** [review]

**Pull request merged**